### PR TITLE
Adding backup lib

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Add backup libs for timeboard and screenboard
Fix images size

### CODE REVIEW

File: content/graphing/dashboards/screenboard.md [hunk 1]
```diff
@@ -60,15 +61,16 @@ This feature can be enabled by following these simple steps:
 This Chrome plug-in helps you change the Color Scheme for your browser (Contrast, Reverse Colors, Image colors, Luminance) [Deluminate](https://chrome.google.com/webstore/detail/deluminate/iebboopaeangfpceklajfohhbpkkfiaa?hl=en-US).
 
 Simple to setup and easily activated/unactivated, Deluminate is a good fit for Datadog Screenboards.
-{{< img src="developers/faq/deluminate.png" alt="deluminate" responsive="true" popup="true">}}
+
+{{< img src="developers/faq/deluminate.png" alt="deluminate" responsive="true" popup="true" style="width:50%;">}}
 
 ### Stylebot (Chrome)
 
 Stylish offers more possibilities to customize the css of any web page.
```
Comment (technovangelist) [inline content/graphing/dashboards/screenboard.md]: I realize you didn't add this, but the title is stylebot and the text says stylish

**Comment by Martiflex** [comment]
Seems good to me

File: content/graphing/dashboards/screenboard.md (hunk 1 above)
Comment (technovangelist) [reply content/graphing/dashboards/screenboard.md]: but then again, why is this even in here. seems inappropriate to have instructions for a third party software in our docs.

File: content/graphing/dashboards/screenboard.md [hunk 2]
```diff
@@ -83,11 +85,12 @@ Here is how to proceed to enjoy this css style on Chrome using Stylebot:
 
 If you have any custom css style sheet that you would like to share, [reach out to us](/help)!
 
-## Backup my Screenboard
+## Back up my Screenboard
```
Comment (technovangelist) [inline content/graphing/dashboards/screenboard.md]: No, i think the proper term is backup. back up doesn't make sense in this context.

File: content/graphing/dashboards/timeboard.md [hunk 3]
```diff
@@ -46,16 +46,17 @@ https://app.datadoghq.com/event/stream?per_page=30&query=tags:audit%20status:all
...
 1. At the top right corner of a dashboard, click on the gear icon:
-    {{< img src="graphing/dashboards/faq/enable_notifications.png" alt="enable notifications" responsive="true" popup="true">}}
+    {{< img src="graphing/dashboards/faq/enable_notifications.png" alt="enable notifications" responsive="true" popup="true" style="width:30%;">}}
 
 2. Select **Notifications** option and enable the notifications:
-    {{< img src="graphing/dashboards/faq/notifications_pop_up.png" alt=" notifications pop up" responsive="true" popup="true">}}
+    {{< img src="graphing/dashboards/faq/notifications_pop_up.png" alt=" notifications pop up" responsive="true" popup="true" style="width:30%;">}}
 
-## Backup my Timeboard
+## Back up my Timeboard
```
Comment (technovangelist) [inline content/graphing/dashboards/timeboard.md]: again, the term is backup. if you have to separate, at least have the hyphen, but i think backup is better.

**Pull request merged**